contrib/python/podman/podman/libs/containers.py

```diff
@@ -129,21 +126,23 @@ def commit(self,
             USER=bozo:circus
             VOLUME=/data
             WORKDIR=/data/application
+
+        All changes overwrite existing values.
+          See inspect() to obtain current settings.
         """
-        # TODO: Clean up *args, **kwargs after Commit() is complete
-        try:
-            author = kwargs.get('author', getpass.getuser())
-        except Exception:  # pylint: disable=broad-except
-            author = ''
+        author = kwargs.get('author', None) or getpass.getuser()
+        change = kwargs.get('change', None) or []
+        message = kwargs.get('message', None) or ''
+        pause = kwargs.get('pause', None) or True
 
-        for c in changes:
+        for c in kwargs['change']:
```

Comment:
Maybe ``for c in kwargs.get('change', [])``?  Not sure if that's correct for this context or not, just responding to the CI test fail.

Reply:
Too aggressive F&R, should have been 'change'.  The code above will use get() to default change. PR update coming.